modules/installation-osp-api-octavia.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * networking/openstack/load-balancing-openstack.adoc
```

Comment:
The file path should be: networking/load-balancing-openstack.adoc (applies to all modules)

Just a nit, and not a required change. :)